src/main/java/org/isf/opd/model/Opd.java

```diff
@@ -139,6 +139,21 @@
 	@Column(name="OPD_LOCK")
 	private int lock;
 	
+	@Column(name="OPD_REASON")
+    private String reason; // ADDED: Arnaud
+
+	@Column(name="OPD_ANAMNESIS")
+    private String anamnesis; // ADDED: Arnaud
+    
+	@Column(name="OPD_ALLERGIES")
+    private String allergies; // ADDED: Arnaud
+    
+	@Column(name="OPD_THERAPIES")
+    private String therapies; // ADDED: Arnaud
+    
+	@Column(name="OPD_PRESCRIPTION")
+    private String prescription; // ADDED: Arnaud
+	
 	@Transient
```

Comment:
it seems this PR is introducing too many feature changes, I think we should split it into different PR and link them to a Jira issue